src/Node/ReadLine.purs

```diff
@@ -1,42 +1,86 @@
 -- | This module provides a binding to the Node `readline` API.
 
-module Node.ReadLine where
+module Node.ReadLine
+  ( Interface
+  , InterfaceOptions
+  , Completer
+  , LineHandler
+  , createInterface
+  , createInteractiveConsoleInterface
```

Comment:
I went back and forth a few times.  If you feel strongly about it I'll shorten it back to `createConsoleInterface`.  It was mostly because when I first came in and read the code the other day I thought it was just a line splitter for streams.  "Interactive" seemed to communicate the intent more, for building interactive (both input and output) console apps.